File: src/test/java/com/infotarget/codility/java/MissingIntegerTest.java
package com.infotarget.codility.java;

import org.junit.Assert;
import org.junit.Test;

import static org.junit.Assert.*;

public class MissingIntegerTest {

    private MissingInteger missingInteger = new MissingInteger();

    @Test
    public void testSolution() throws Exception {
        Assert.assertEquals(5, missingInteger.solution(new int[]{1, 3, 6, 4, 1, 2}));
    }
}

File: src/test/scala/com/infotarget/codility/scala/MissingIntegerTest.scala
package com.infotarget.codility.scala

import org.scalatest.FunSuite

class MissingIntegerTest extends FunSuite {

  test("testSolution") {
    assert(5 === MissingInteger.solution(Array[Int](1, 3, 6, 4, 1, 2)))
  }

}
